examples/Importers/ImportURDFDemo/UrdfRenderingInterface.h

```diff
@@ -80,12 +86,18 @@ struct UrdfRenderingInterface
 	///provide the image pixels as a part of a stream.
 	virtual void copyCameraImageData(unsigned char* pixelsRGBA, int rgbaBufferSizeInPixels, float* depthBuffer, int depthBufferSizeInPixels,int* segmentationMaskBuffer, int segmentationMaskSizeInPixels,  int startPixelIndex, int* widthPtr, int* heightPtr, int* numPixelsCopied)=0;
 	
+	///provide the camera array image pixels as a part of a stream.
+	virtual void copyCameraArrayImageData(unsigned char* pixelsRGB, int rgbaBufferSizeInPixels, float* featuresBuffer, int featuresBufferSizeInFloats, int *cameraArraySizePtr, int* widthPtr, int* heightPtr, int *featureLength, int* numPixelsCopied, int* numFeaturesCopied)=0;
+
 	///render an image, using some arbitraty view and projection matrix
 	virtual void render()=0;
 	
 	///render an image using the provided view and projection matrix
 	virtual void render(const float viewMat[16], const float projMat[16])=0;
-	
+
+	///render an image using the provided view and projection matrix
+	virtual void renderCameraArray(const float (*viewMat)[16], const float (*projMat)[16]) = 0;
```

Comment:
How about removing get/setCameraArraySize and passing the number of cameras in renderCameraArray? Fewer new methods is better.